protobuf/protoc-gen-govalidator/valtest/valtest_test.go

```diff
@@ -112,7 +112,7 @@ var valMsg = ValTestMessage{
 	AllString:             " Lot of checks here>",
 	Name:                  "Sinéad O'Connor",
 	NoValidation:          "<really?>' OR 1=1",
-	ContactOneof:          &ValTestMessage_Phone{Phone: "145-456.123"},
+	ContactOneof:          &ValTestMessage_Phone{Phone: "14574560123"},
```

Comment:
Previous, we don't support `oneof` so there was no validation for Phone. Since `oneof` is now supported, I'll need to update the phone number to pass the tests.